projects/schematics/src/shared/lib-configs/integration-libs/opf-schematics-config.ts

```diff
@@ -0,0 +1,272 @@
+export const OPF_PAYMENT_MODULE = 'OpfPaymentModule';
+export const OPF_PAYMENT_ROOT_MODULE = 'OpfPaymentRootModule';
+export const OPF_PAYMENT_TRANSLATIONS = 'opfPaymentTranslations';
+export const OPF_PAYMENT_TRANSLATION_CHUNKS_CONFIG =
+  'opfPaymentTranslationChunksConfig';
+
+export const OPF_BASE_SCHEMATICS_CONFIG: SchematicConfig = {
+  library: {
+    featureName: OPF_BASE_FEATURE_NAME,
+    mainScope: SPARTACUS_OPF,
+    featureScope: SPARTACUS_OPF_BASE,
+  },
+  folderName: OPF_FOLDER_NAME,
+  moduleName: OPF_MODULE_NAME,
+  featureModule: {
+    name: OPF_BASE_MODULE,
+    importPath: SPARTACUS_OPF_BASE,
+  },
+  rootModule: {
+    name: OPF_BASE_ROOT_MODULE,
+    importPath: SPARTACUS_OPF_BASE_ROOT,
+  },
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_OPF_BASE_ROOT,
+    namedImports: [OPF_BASE_FEATURE_NAME_CONSTANT],
+  },
+  styles: {
+    scssFileName: OPF_SCSS_FILE_NAME,
+    importStyle: SPARTACUS_OPF,
+  },
+  customConfig: buildOpfConfig,
+};
+
+export const OPF_PAYMENT_SCHEMATICS_CONFIG: SchematicConfig = {
+  library: {
+    featureName: OPF_PAYMENT_FEATURE_NAME,
+    mainScope: SPARTACUS_OPF,
+    featureScope: SPARTACUS_OPF_PAYMENT,
+  },
+  folderName: OPF_FOLDER_NAME,
+  moduleName: OPF_MODULE_NAME,
+  featureModule: {
+    name: OPF_PAYMENT_MODULE,
+    importPath: SPARTACUS_OPF_PAYMENT,
+  },
+  rootModule: {
+    name: OPF_PAYMENT_ROOT_MODULE,
+    importPath: SPARTACUS_OPF_PAYMENT_ROOT,
+  },
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_OPF_PAYMENT_ROOT,
+    namedImports: [OPF_PAYMENT_FEATURE_NAME_CONSTANT],
+  },
+  i18n: {
+    resources: OPF_PAYMENT_TRANSLATIONS,
+    chunks: OPF_PAYMENT_TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_OPF_PAYMENT_ASSETS,
+  },
+  styles: {
+    scssFileName: OPF_SCSS_FILE_NAME,
+    importStyle: SPARTACUS_OPF,
+  },
+  customConfig: buildOpfConfig,
+};
+
+export const OPF_CHECKOUT_SCHEMATICS_CONFIG: SchematicConfig = {
+  library: {
+    featureName: OPF_CHECKOUT_FEATURE_NAME,
+    mainScope: SPARTACUS_OPF,
+    featureScope: SPARTACUS_OPF_CHECKOUT,
+  },
+  folderName: OPF_FOLDER_NAME,
+  moduleName: OPF_MODULE_NAME,
+  featureModule: {
+    name: OPF_CHECKOUT_MODULE,
+    importPath: SPARTACUS_OPF_CHECKOUT,
+  },
+  rootModule: {
+    name: OPF_CHECKOUT_ROOT_MODULE,
+    importPath: SPARTACUS_OPF_CHECKOUT_ROOT,
+  },
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_OPF_CHECKOUT_ROOT,
+    namedImports: [OPF_CHECKOUT_FEATURE_NAME_CONSTANT],
+  },
+  i18n: {
+    resources: OPF_CHECKOUT_TRANSLATIONS,
+    chunks: OPF_CHECKOUT_TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_OPF_CHECKOUT_ASSETS,
+  },
+  styles: {
+    scssFileName: OPF_SCSS_FILE_NAME,
+    importStyle: SPARTACUS_OPF,
+  },
+  customConfig: buildOpfConfig,
+  dependencyFeatures: [
+    OPF_PAYMENT_FEATURE_NAME,
+    OPF_BASE_FEATURE_NAME,
+    OPF_CTA_FEATURE_NAME,
+    OPF_GLOBAL_FUNCTIONS_FEATURE_NAME,
+    OPF_QUICK_BUY_FEATURE_NAME,
+  ],
```

Comment:
[QUESTION]
Do we really need those `dependencyFeatures` being declared here?
If yes, then why don't we declare them for other sections like OPF_CTA_SCHEMATICS_CONFIG?

I guess it's a "trick" to install all features at once under the "umbrella" of the OPF Checkout, if a customer chooses to install OPF Checkout. And I can see in the main installation schematics (for `ng add spartacus...`, we have just one option to choose for the whole OPF - `OPF-Checkout`, see: https://github.com/SAP/spartacus/blob/d0b209dae77cd503ed429e3dad1bb46ebfe0af19/projects/schematics/src/add-spartacus/schema.json#L37

Then let's add a code comment here, explaining the reason of listing dependency feature here (but not in other places).